doc/_includes/tutorials/common/samples/copy-samples.md

```diff
@@ -19,15 +19,21 @@ Once selected, you will be redirected to the new Share / Move Samples page.
 
 The first action is to select the destination project. Start typing the name of the destination project in the "Select a
 project to share samples with" input. As you type a list of projects will be presented, select the project you want.
+Once you select a project to share with, the "Next" button will become enabled, clicking on it will allow you to review
+the samples that were selcted to copy.
```

Comment:
Typo in `selected`

Reply:
Updated in 0e8f9a2a60ad5a105bb12339b85daf89fe3908f1